specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-02-14-privatepreview/examples/ServerCreate.json

```diff
@@ -0,0 +1,90 @@
+{
+  "parameters": {
+    "serverName": "pgtestsvc4",
+    "resourceGroupName": "TestGroup",
+    "api-version": "2020-02-14-privatepreview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "parameters": {
+      "location": "westus",
+      "properties": {
+        "administratorLogin": "cloudsa",
+        "administratorLoginPassword": "password",
+        "serverEdition": "GeneralPurpose",
+        "vCores": 4,
+        "storageQuotaInMb": 524288,
+        "version": "12",
+        "standbyCount": 0,
```

Comment:
userVisibleHAState and userVisibleState properties seem to be missing. Is that on purpose?

Reply:
need to discuss further with API contratc level, last time we got a comment to convert the userVisibleState to state thats why i coded it as state. Need to check with Xin whether it is updated or not.

Reply:
Let's keep userVisibleState -- this is to align with version 2017-12-01. We'd like to our customer to make minimal changes to their automation.

Reply:
added userVisibleHAState, userVisibleState